integration_tests/exemplars_otel_agent/src/test/resources/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:11-jre
+RUN mkdir -p /app/
+ADD https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.1.0/opentelemetry-javaagent-all.jar /app/
```

Comment:
1.2.0?

Reply:
Seems that the agent release lacks a few days behind the library release. Agent is still 1.1.0.